README.rst

```diff
@@ -94,6 +123,38 @@ Contributing
 
 See https://docs.scrapy.org/en/master/contributing.html for details.
 
+If you are a new contributor, you may benefit from starting at the following locations. This will provide you
```

Comment:
https://docs.scrapy.org/en/master/contributing.html linked in the previous line already has a ToC.